app/components/Button/index.js

```diff
@@ -7,37 +7,64 @@
  */
 
 import React, { PropTypes, Children } from 'react';
+import styled from 'styled-components';
 
-import styles from './styles.css';
+const buttonStyling = `
```

Comment:
Use `css` for the `buttonStyles` like so:

  ```JS
  import styled, { css } from 'styled-components';

  const buttonStyles = css`
    styles: here;
  `;
  ```

Otherwise if people add interpolated functions here they won't work!